regression/acceleration/Makefile

```diff
@@ -28,5 +28,6 @@ show:
 	done;
 
 clean:
-	$(RM) tests.log
-	$(RM) */main.out
+	find -name '*.log' -execdir $(RM) '{}' \;
+	find -name '*.gb' -execdir $(RM) '{}' \;
+	find -name '*.out' -execdir $(RM) '{}' \;
```

Comment:
In other cases you explicitly added the `.` after `find`. Both might work, but please be consistent.

Reply:
Thanks, I missed some in adding the `.`.